users/views.py

```diff
@@ -1,3 +1,95 @@
-from django.shortcuts import render
+import json, re, bcrypt, jwt
+from .models            import User, Address
+from django.db.models   import Q
+from django.http        import JsonResponse
+from django.views       import View
 
-# Create your views here.
+class UserView(View):
+    def post(self, request):
+
+        try:
+            data                 = json.loads(request.body)
+            identification       = data['id']
+            password             = data['password']
+            name                 = data['name']
+            email                = data['email']
+            phone_number         = data['phone_number']
+            birth_date           = data['birthdate']
+            gender               = data['gender']
+            address              = data['address']
+            phone_check          = re.compile('^[0-9]{3}[0-9]{3,4}[0-9]{4}')
+            identification_check = re.compile('^[0-9a-zA-Z]{6,16}$')
+            password_check       = re.compile('^(?=.*[a-zA-Z0-9])(?=.*[a-zA-Z!@#$%^&*])(?=.*[0-9!@#$%^&*]).{10,16}$')
+            email_check          = re.compile('^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$')
```

Comment:
```suggestion
            phone_check          = re.compile('^[0-9]{3}[0-9]{3,4}[0-9]{4}')
            identification_check = re.compile('^[0-9a-zA-Z]{6,16}$')
            password_check       = re.compile('^(?=.*[a-zA-Z0-9])(?=.*[a-zA-Z!@#$%^&*])(?=.*[0-9!@#$%^&*]).{10,16}$')
            email_check          = re.compile('^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$')
```
이 요소들은 my_settings에 상수로 저장해서 임포트 해서 사용해주세요.

Reply:
수정하겠습니다~